data/presets/mountain_pass.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "temaki-saddle",
+    "fields": [
+        "name",
+        "ele_node"
+    ],
+    "geometry": [
+        "point",
```

Comment:
```suggestion
```

I've only seen `mountain_pass=yes` mapped as a vertex. If there's no road (or path) going through, a saddle is just a saddle, isn't it?